lib/validator.js

```diff
@@ -78,9 +78,7 @@ Validator.prototype._convertToJsonSchema = function(parameters) {
                 // 'required' array must have at least one element according to json-schema spec,
                 // se we can't preinitialize it.
                 schema.required = schema.required || [];
-                if (schema.required.indexOf(inMapping[param.in]) < 0) {
-                    schema.required.push(inMapping[param.in]);
-                }
+                schema.required.push(inMapping[param.in]);
```

Comment:
@gwicke Yes. The previous code was dead code. We can get here only if `!schema.properties[inMapping[param.in]]` (line 73). And if that condition is true, we immediately set this property (line 74). So uniqueness of the array is already ensured without this if.